src/nvim/api/vim.c

```diff
@@ -85,6 +86,54 @@ void api_vim_free_all_mem(void)
   map_free(String, handle_T)(namespace_ids);
 }
 
+void nvim_source(String command, Error *err) FUNC_API_SINCE(7)
```

Comment:
Missing docstring. Doesn't need be long, just something like "execute a multiline block of ex commands from a string" and list the args.

`command` arg should be renamed though, at it is probably not a single command. Maybe `src` ?

Reply:
Have added the docstrings, and renamed the arguments.